.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file.  I believe it's meant for use with Eclipse and not meant to be shared.